juju/juju
# Deploying applications in development mode

## Description of change
This add a `--develop` flag to the deploy command. The flag registers the application as being in development mode both with an outside API call and on state.
Later the fact that an application is in `development mode` will be used to gate access to things like meter status overrides and metrics debugging.

## QA steps

Run `juju deploy wordpress --develop`. For the time being this should result in an error (server side not supported).

## Documentation changes

Documentation for the `deploy` command will need to be updated. Check with @cmars regarding the details.
